apps/nextjs/src/app/(dashboard)/dashboard/_components/sidebar/nav-menu-switcher.tsx

```
@@ -0,0 +1,73 @@
+      <div className="flex aspect-square size-8 items-center justify-center rounded-lg text-sidebar-primary-foreground">
+        <BattleStadiumLogoLink />
+      </div>
+
+      <SidebarMenu>
+        <SidebarMenuItem>
+          <DropdownMenu>
+            <DropdownMenuTrigger asChild>
+              <SidebarMenuButton
+                size="lg"
+                aria-label="Menu with links to main app"
+                role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)